src/components/Popups/PopupDrawingToolbar.tsx

```diff
@@ -50,16 +53,24 @@ const PopupDrawingToolbar = ({
     canRedo,
     canUndo,
     className,
+    drawnPathGroups,
     onDelete,
     onRedo,
     onReply,
     onUndo,
     reference,
 }: Props): JSX.Element => {
     const intl = useIntl();
+    const popupRef = React.useRef<PopupBase>(null);
+
+    React.useEffect(() => {
+        const { current: popup } = popupRef;
+        popup.popper.update();
+    }, [popupRef, drawnPathGroups]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Is this eslint-disable needed? Can we alphabetize the deps, as well?

Reply:
Not needed, alphabetized.